From 96826042b5a5546cb3c0aafde1d89529d738b7f2 Mon Sep 17 00:00:00 2001
From: Caesar Wang <wxt@rock-chips.com>
Date: Tue, 1 Dec 2020 07:53:11 +0000
Subject: [PATCH] arm64: configs/rockchip: update rockchip_linux_defconfig

The below functions are updated for rockchip_linux_defconfig:

1) enable iep2
2) enable ov5695 and ov7251
3) enable isp for isp2x
4) remove unused SCSI

Signed-off-by: Caesar Wang <wxt@rock-chips.com>
Change-Id: I74c91c7c0960f2d65cfd0b70e55fbc964deedcec
---
 arch/arm64/configs/rockchip_linux_defconfig | 5 ++++-
 1 file changed, 4 insertions(+), 1 deletion(-)

diff --git a/arch/arm64/configs/rockchip_linux_defconfig b/arch/arm64/configs/rockchip_linux_defconfig
index bb02e0302d76..09395f8c28e9 100644
--- a/arch/arm64/configs/rockchip_linux_defconfig
+++ b/arch/arm64/configs/rockchip_linux_defconfig
@@ -113,7 +113,6 @@ CONFIG_BLK_DEV_RAM=y
 CONFIG_BLK_DEV_RAM_COUNT=1
 CONFIG_BLK_DEV_NVME=y
 CONFIG_SRAM=y
-CONFIG_SCSI=y
 CONFIG_BLK_DEV_SD=y
 CONFIG_BLK_DEV_SR=y
 CONFIG_SCSI_SCAN_ASYNC=y
@@ -281,11 +280,14 @@ CONFIG_USB_VIDEO_CLASS=y
 CONFIG_V4L_PLATFORM_DRIVERS=y
 CONFIG_SOC_CAMERA=y
 CONFIG_VIDEO_ROCKCHIP_ISP1=y
+CONFIG_VIDEO_ROCKCHIP_ISP=y
 CONFIG_V4L_MEM2MEM_DRIVERS=y
 CONFIG_VIDEO_ROCKCHIP_RGA=y
 # CONFIG_MEDIA_SUBDRV_AUTOSELECT is not set
 CONFIG_VIDEO_TC35874X=y
 CONFIG_VIDEO_OV4689=y
+CONFIG_VIDEO_OV5695=y
+CONFIG_VIDEO_OV7251=y
 CONFIG_VIDEO_OV13850=y
 # CONFIG_VGA_ARB is not set
 CONFIG_DRM=y
@@ -328,6 +330,7 @@ CONFIG_ROCKCHIP_MPP_VDPU1=y
 CONFIG_ROCKCHIP_MPP_VEPU1=y
 CONFIG_ROCKCHIP_MPP_VDPU2=y
 CONFIG_ROCKCHIP_MPP_VEPU2=y
+CONFIG_ROCKCHIP_MPP_IEP2=y
 CONFIG_ROCKCHIP_MPP_JPGDEC=y
 CONFIG_SOUND=y
 CONFIG_SND=y
-- 
2.35.3

